src/cli/repl/core.ts

```diff
@@ -11,13 +11,40 @@ import { commandNames, getCommand, standardReplOutput } from './commands'
 import * as readline from 'node:readline'
 import { splitAtEscapeSensitive } from '../../util/args'
 import { executeRShellCommand } from './commands/execute'
+import { guard } from '../../util/assert'
+import { scripts } from '../common/scripts-info'
+import type { OptionDefinition } from 'command-line-usage'
 
 const replCompleterKeywords = Array.from(commandNames, s => `:${s}`)
 
 /**
  * Used by the repl to provide automatic completions for a given (partial) input line
  */
 export function replCompleter(line: string): [string[], string] {
+	const singleCommand = replCompleterKeywords.find(k => (k + ' ') === line)
+	if(singleCommand !== undefined){
+		const applicableCommand = getCommand(singleCommand.substring(1))
+		guard(applicableCommand !== undefined, 'Command should be defined')
+		if(applicableCommand.script){
+			let scriptOptions: OptionDefinition[] | undefined
+			for(const [script, { options}] of Object.entries(scripts)){
+				if(script === line.trimEnd().substring(1)){
+					scriptOptions = options
+					break
+				}
+			}
+			guard(scriptOptions !== undefined, 'script should be in script record')
+			const autoCompleteList : string[] = []
+			autoCompleteList.push(line)
+			scriptOptions.forEach(option => {
+				autoCompleteList.push(line + ' --' + option.name)
```

Comment:
wir müssen noch schauen, dass wir es so hinkriegen, dass man auch mehrere Optionen angeben kann usw. Da gibt es bestimmt schon eine Bibliothek oder eine Möglichkeit das mit readline hinzubekommen. Kannst du da mal schauen?

Reply:
Meinst du jetzt eine Permutation der Kommandos? Sowas wie: Wenn es -a und -b für command gibt, dass dann in der liste [command -a, command -b , command -a -b] steht?

Reply:
ich mein, dass man auch bei `foo --bar --baz` noch weiter completen kann, aber dann nur die Optionen kriegt die auch mehrfach gehen. Es sollte da auch sicher pakete geben (oder: https://stackoverflow.com/questions/44708200/nodejs-readline-autocomplete-several-words)